NRLMMD-GEOIPS/geoips_clavrx
# 22 reader unit test

# Reviewer Checklist

PR author: Please ensure you meet all of the below requirements, and check boxes appropriately.

Reviewers: Please confirm all required testing/documentation has been completed prior to approving.

**Remove lines that are not applicable, explain if you select "NO REQUIRED"**


* [x] Required ***unit tests*** added and pass for new/modified functionality



https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md


# Testing Instructions
Run pytest on following code, verify that it generates 1 pass and 1 xfail:
```geoips_clavrx/tests/unit_tests/plugins/modules/readers/clavrx_reader_test.py```

# Summary
* Created unit testing for clavrx reader testing geoips xarray conformity 

# Output
```
================================================== test session starts ===================================================
platform linux -- Python 3.10.12, pytest-7.4.2, pluggy-1.3.0
rootdir: /home/ajthorp
plugins: anyio-4.0.0, cov-4.1.0
collected 2 items

clavrx_reader_test.py .x                                                                                           [100%]

======================================== 1 passed, 1 xfailed in 93.92s (0:01:33) =========================================
```
